store/tikv/region_cache.go

```diff
@@ -269,6 +269,7 @@ type RPCContext struct {
 	PeerIdx int
 	Store   *Store
 	Addr    string
+	User    *kv.UserIdentity
```

Comment:
Why `User` need to be a part of `RPCContext`?
The coprocessor request itself can contain this information?

Reply:
Great~